library/src/f3d_interactor_impl.cxx

```diff
@@ -460,7 +460,7 @@ bool interactor_impl::recordInteraction(const std::string& file)
   if (file.empty())
   {
     // TODO improve VTK to check file opening
-    f3d::log::error("Interaction record file is empty");
+    log::error("Interaction record file is empty");
```

Comment:
shouldnt this line be covered by testing already ?

Reply:
I don't think so